meteor/client/ui/RundownView/RundownViewShelf.tsx

```diff
@@ -0,0 +1,280 @@
+import * as React from 'react'
+import * as _ from 'underscore'
+import { Translated, translateWithTracker } from '../../lib/ReactMeteorData/ReactMeteorData'
+import { MeteorReactComponent } from '../../lib/MeteorReactComponent'
+import { SegmentUi } from '../SegmentTimeline/SegmentTimelineContainer'
+import { normalizeArray, unprotectString } from '../../../lib/lib'
+import { RundownPlaylist } from '../../../lib/collections/RundownPlaylists'
+import { ShowStyleBase } from '../../../lib/collections/ShowStyleBases'
+import { DashboardPieceButton } from '../Shelf/DashboardPieceButton'
+import { IBlueprintActionTriggerMode, IOutputLayer, ISourceLayer } from '@sofie-automation/blueprints-integration'
+import { Studio } from '../../../lib/collections/Studios'
+import { ensureHasTrailingSlash, UserAgentPointer, USER_AGENT_POINTER_PROPERTY } from '../../lib/lib'
+import { PieceDisplayStyle } from '../../../lib/collections/RundownLayouts'
+import { NoticeLevel, Notification, NotificationCenter } from '../../lib/notifications/notifications'
+import { memoizedIsolatedAutorun } from '../../lib/reactiveData/reactiveDataHelper'
+import { PartInstanceId } from '../../../lib/collections/PartInstances'
+import { PieceId } from '../../../lib/collections/Pieces'
+import { doUserAction, UserAction } from '../../lib/userAction'
+import { MeteorCall } from '../../../lib/api/methods'
+import {
+	AdLibPieceUi,
+	AdlibSegmentUi,
+	getNextPieceInstancesGrouped,
+	getUnfinishedPieceInstancesGrouped,
+	isAdLibNext,
+	isAdLibOnAir,
+} from '../../lib/shelf'
+
+interface IRundownViewShelfProps {
+	studio: Studio
+	segment: SegmentUi
+	playlist: RundownPlaylist
+	showStyleBase: ShowStyleBase
+	adLibSegmentUi: AdlibSegmentUi
+	hotkeyGroup: string
+	studioMode: boolean
+	registerHotkeys?: boolean
```

Comment:
This doesn't seem to be used anywhere or do anything.

Reply:
Leftovers from pre-actionTriggers times. 
(But that also means we seem to have a lost feature here, because it was supposed to show the assigned shortcut on the button)